sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -119,6 +120,20 @@ struct {
       ext::oneapi::level_zero::ownership::transfer};
 }
 ```
+
+Deprecated.[^1]

```

Comment:
We should make it clear that the API changed in version 3 of the specification.  Maybe say "Deprecated as of version 3 of this specification".

Down below for the new API, we should say something like "This version of the structure supported starting with version 3 of this specification".

Reply:
I have added clarifications and made them share the footnote as I think it is applicable to both. Is this in line with what you were thinking?